server/datastore/mysql/hosts.go

```diff
@@ -442,28 +446,41 @@ func loadHostScheduledQueryStatsDB(ctx context.Context, db sqlx.QueryerContext,
 			SUM(stats.wall_time) AS wall_time
 		FROM scheduled_query_stats stats WHERE stats.host_id = ? GROUP BY stats.scheduled_query_id) as sqs ON (q.id = sqs.scheduled_query_id)
 		LEFT JOIN query_results qr ON (q.id = qr.query_id AND qr.host_id = ?)
+	`
+
+	filter1 := `
 		WHERE
 			(q.platform = '' OR q.platform IS NULL OR FIND_IN_SET(?, q.platform) != 0)
 			AND q.schedule_interval > 0
 			AND (q.automations_enabled IS TRUE OR (q.discard_data IS FALSE AND q.logging_type = ?))
```

Comment:
Have you tested with randomized `discard_data` and `logging_type` entries in DB? Maybe they also need to be part of the index.

Reply:
tested some filter varieties on those columns and haven't seen any degradation in performance.  Still using the new index.